deployment/cdk/opensearch-service-migration/lib/common-utilities.ts

```diff
@@ -396,7 +421,7 @@ export function parseClusterDefinition(json: any): ClusterYaml {
     }
     const auth = parseAuth(json.auth)
     if (!auth) {
-        throw new Error(`Invalid auth type when parsing cluster definition: ${json.auth.type}`)
+        throw new Error(`Invalid auth type when parsing cluster definition: ${json.auth}`)
```

Comment:
Can we hold on this change until the plaintext password option is removed from this?